hw1/README.rst

```diff
@@ -0,0 +1,56 @@
+======================
```

Comment:
You should not include the files that I added.  Please only add files in your own directory.  Otherwise this PR cannot be merged.

Reply:
sorry, I dont know how to cancel this commit. I tried searching for this situation, but the search results were unsatisfactory (git commit --amend). Could you tell me what can I do.

Reply:
Please take a look at the man page of git, especially the command commit and squash.  You may reach the grader and other classmates for more information.